roomserver/api/perform.go

```diff
@@ -0,0 +1,59 @@
+package api
+
+import (
+	"context"
+
+	commonHTTP "github.com/matrix-org/dendrite/common/http"
+	"github.com/opentracing/opentracing-go"
+)
+
+const (
+	// RoomserverPerformJoinRequestPath is the HTTP path for the PerformJoinRequest API.
+	RoomserverPerformJoinRequestPath = "/api/roomserver/performJoin"
+
+	// RoomserverPerformLeaveRequestPath is the HTTP path for the PerformLeaveRequest API.
+	RoomserverPerformLeaveRequestPath = "/api/roomserver/performLeave"
```

Comment:
Probably shouldn't include this stuff...

Reply:
Oh, whoops. That was in my tree from another branch.